documentation/API.md

```diff
@@ -222,6 +222,27 @@ returns:
 }
 ```
 
+#### Generate PDF
+
+POST: `/generate-pdf`
+
+Endpoint to generate PDF files based on a [Carbone template](https://carbone.io/api-reference.html).
+
+**Parameters** (as body JSON):
+
+- `fileId` -- uniqueId of the carbone template file (from the "file" table)
+- `data` -- object containing all the data to be used for substitutions in the carbone template
```

Comment:
It does do that, but I guess it's a little confusing having both "data" and "additionalData" parameters, since they basically do the same thing.
```
const data = parameters?.data ?? { ...applicationData, ...outputCumulative, additionalData }
```

I could just pretend the "data" parameter doesn't exist (leave it out of docs) and just use it internally?

Reply:
Yes, I think that would be better indeed

Reply:
Cool, done: 5caf6dc8

I think that makes it clearer